osu.Game/Overlays/MusicController.cs

```diff
@@ -181,6 +183,35 @@ private bool next(bool instant = false)
             return false;
         }
 
+        private bool restartTrack()
```

Comment:
You can use an out parameter instead of a private field that is seemingly misleading
```suggestion
        private bool restartTrack(out bool restarted)
```

Reply:
what.

book return and bool our with no documentation this is shocking.

Reply:
Forgot to mention that